src/vcUnitConversion.cpp

```diff
@@ -0,0 +1,174 @@
+#include "vcUnitConversion.h"
+
+double vcUnitConversion_ConvertDistance(double sourceValue, vcDistanceUnit sourceUnit, vcDistanceUnit requiredUnit)
+{
+  double scalar = 1.0;
+
+  // Metric
+  if (sourceUnit == vcDistance_Metres)
+    scalar = 1.0;
+  else if (sourceUnit == vcDistance_Kilometres) // If source unit is in Meter than Multiply by 1000 to get kilometer.
+    scalar = 1000;
```

Comment:
This should end with `.0`, similarly with all the other doubles in this file that are whole numbers.